ios/FluentUI.Demo/FluentUI.Demo/Demos/PillButtonBarDemoController.swift

```diff
@@ -82,8 +82,13 @@ class PillButtonBarDemoController: DemoController {
         }
     }
 
-    func createBar(items: [PillButtonBarItem], style: PillButtonStyle = .outline, centerAligned: Bool = false, disabledItems: Bool = false, useCustomPillsBackground: Bool = false) -> UIView {
-        let bar = PillButtonBar(pillButtonStyle: style, pillButtonBackgroundColor: useCustomPillsBackground ? Colors.Palette.communicationBlueShade30.color : nil)
+    func createBar(items: [PillButtonBarItem], style: PillButtonStyle = .outline, centerAligned: Bool = false, disabledItems: Bool = false, useCustomPillsColors: Bool = false) -> UIView {
```

Comment:
Can we make sure that the colors used in the demo app adhere to contrast ratio requirements? You can check [here](https://webaim.org/resources/contrastchecker/).